### PR TITLE
fix(Link/react): pass props

### PR DESCRIPTION
**Pull request by krishnaacc123**

### CODE REVIEW

**Comment by chaitanyadeorukhkar** [comment]
We shouldn't do this. This opens up the possibility of passing any props including style props that will break all the design restrictions

**Comment by krishnaacc123** [comment]
@chaitanyadeorukhkar But then how will we make custom components out of this with particular css at the consumer side.

File: src/atoms/Link/Link.web.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ const StyledText = styled(Text)`
   }
 `;
 
-const Link = ({ size, children, href, target, testID, rel, disabled }) => {
+const Link = ({ size, children, href, target, testID, rel, disabled, ...props }) => {
```
Comment (kamleshchandnani) [inline src/atoms/Link/Link.web.js:53]: Give some reasoning behind this?

Comment (krishnaacc123) [reply src/atoms/Link/Link.web.js:53]: We are using custom components out of this with particular css at the consumer side.

**Comment by kamleshchandnani** [comment]
As discussed closing this

**Pull request closed without merging**